coalib/results/TextPosition.py

```diff
@@ -9,8 +9,8 @@ class TextPosition:
     @enforce_signature
     def __init__(self, line: (int, None)=None, column: (int, None)=None):
         """
-        Creates a new TextPosition object that represents the position inside
-        a string with line/column numbers.
+        Creates a new ``TextPosition`` object that represents the position
+        inside a string with line/column numbers.
```

Comment:
removed this change.. I'll update this in other PR (the one implementing _lt_, __gt__)